tests/unit/amo/components/TestWrongPlatformWarning.js

```diff
@@ -42,34 +39,33 @@ describe(__filename, () => {
     });
   };
 
-  const render = ({ location = createFakeLocation(), ...customProps } = {}) => {
+  const render = ({ location, ...customProps } = {}) => {
     const props = {
       _correctedLocationForPlatform,
       _getClientCompatibility,
       _isFirefoxForAndroid,
       _isFirefoxForIOS,
       _isAndroidInstallable,
-      i18n: fakeI18n(),
-      store,
       ...customProps,
     };
+    const renderOptions = {
+      history: createHistory({
+        initialEntries: [location || '/'],
```

Comment:
should `location` default be `'/some/location/'`?  It seems to be what's specified when a location is needed.

Reply:
Actually, `/some/location/` is never passed into the `location` argument of `render`. It's used as the mocked return value of `_correctedLocationForPlatform` in a few places, but it has nothing to do with the `location` in `render`. The only time we care about the `location` in `render` is when we are checking the arguments that `_correctedLocationForPlatform` was called with (for example in "it('calls _correctedLocationForPlatform with isHomePage defaulted to false'"), so there's really no need for a default value other than `/` here.